### PR TITLE
Update metadata for new Vox Pupuli owner. Add license file

### PR DESCRIPTION
**Pull request by nmaludy**
#### Pull Request (PR) description
* Change the owner and links in `metadata.json` to Vox Pupuli.
* Added a license file 
* Updated README badges

### CODE REVIEW

File: metadata.json [hunk 1]
```diff
@@ -1,12 +1,12 @@
 {
-  "name": "nmaludy-erlang",
+  "name": "puppet-erlang",
   "version": "0.1.0",
-  "author": "nmaludy",
+  "author": "puppet",
```
Comment (wyardley) [inline metadata.json:4]: I think this should be “Vox Pupuli” maybe

File: metadata.json [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
   "license": "Apache-2.0",
-  "source": "https://github.com/nmaludy/puppet-erlang",
-  "project_page": "https://github.com/nmaludy/puppet-erlang",
-  "issues_url": "https://github.com/nmaludy/puppet-erlang/issues",
+  "source": "https://github.com/voxpupuli/puppet-erlang",
+  "project_page": "https://github.com/voxpupuli/puppet-erlang",
+  "issues_url": "https://github.com/voxpupuli/puppet-erlang/issues",
   "dependencies": [
     {
       "name": "puppetlabs/apt",
```
Comment (wyardley) [inline metadata.json:12]: I _think_ deps like this that are OS specific are supposed to be soft deps. But it’s been a bit

Comment (wyardley) [reply metadata.json:12]: https://puppet.com/docs/puppet/latest/style_guide.html#dependencies

> A soft dependency is a dependency that is only required in a specific set of use cases

That said, I have seen other modules require OS specific package managers.

File: metadata.json (hunk 1 above)
Comment (ghoneycutt) [reply metadata.json:4]: ```"author": "Vox Pupuli",```

File: metadata.json (hunk 2 above)
Comment (ghoneycutt) [reply metadata.json:12]: I think the dependencies are good as they are.

**Pull request merged**